test/full-info-test.js

```diff
@@ -39,6 +40,27 @@ describe('ytdl.getInfo()', () => {
     });
   });
 
+  describe('With IPv6 Block', () => {
+    it('Sends request with IPv6 address', async() => {
+      const id = '_HSylqgVYQI';
+      const scope = nock(id, 'regular');
+      let info = await ytdl.getInfo(id, { IPv6Block: '2001:2::/48' });
+      nock.url(info.formats[0].url).reply(function checkAddr() {
+        // "this" is assigned by the function checkAddr
+        // eslint-disable-next-line no-invalid-this
+        assert.ok(net.isIPv6(this.req.options.localAddress));
+        scope.done();
+      });
+    });
+  });
+
+  describe('With invalid IPv6 Block', () => {
+    it('Should give an error', async() => {
+      const id = '_HSylqgVYQI';
+      await assert.rejects(ytdl.getInfo(id, { IPv6Block: '2001:2::/200' }), /Invalid IPv6 subnet/);
+    });
+  });
```

Comment:
we could test the `getRandomIPv6` (but not here) function instead of doing the same `pass an ipv6` test (like above) twice

so that we do
a) test the whole usage via ytdl-core once
b) do white box tests only on the `getRandomIPv6` function for error like
  b.a) no subnet (`2001:2::`) or 
  b.b) subnet only (`/64`) or
  b.c) passing errors from `ip6.randomSubnet`